src/logics/stargate/tokens/index.ts

```diff
@@ -3,34 +3,40 @@ import avalancheTokensJSON from './data/avalanche.json';
 import baseTokensJSON from './data/base.json';
 import * as common from '@protocolink/common';
 import mainnetTokensJSON from './data/mainnet.json';
-import metisTokensJSON from './data/metis.json';
```

Comment:
How about continuing to import the empty `metis.json` and also declaring type `MetisTokenSymbols` to maintain consistency with other chains?

Reply:
makes sense, let's maintain consistency.